pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -31,23 +31,44 @@
             </strong>
           </p>
           <% } %>
-          <form id="confirm-form" name="confirm-form" method="POST">
+
+          <% if (locals.assessment.require_honor_code) { %>
+          <div class="card card-secondary mb-4 test-class-honor-code">
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item py-2">I certify that I am <%= user.name %> and I am allowed to take this assessment in <%= course.short_name %> (<%= course_instance.short_name %>).</li>
```

Comment:
```suggestion
              <li class="list-group-item py-2">I certify that I am <%= user.name %> and I am allowed to take this assessment.</li>
```

Naming the course again felt a bit redundant, given that it appears directly above this card. I don't think it's necessary to name the course instance.